tests/PHPStan/Reflection/Annotations/DeprecatedAnnotationsTest.php

```diff
@@ -131,4 +131,15 @@ public function testNonDeprecatedNativeFunctions(): void
 		$this->assertFalse($reflectionProvider->getFunction(new Name('function_exists'), null)->isDeprecated()->yes());
 	}
 
+	public function testDeprecatedMethodsFromInterface(): void
+	{
+		require_once __DIR__ . '/data/annotations-deprecated.php';
```

Comment:
The require doesn't need to be there. You just need to make sure to run `composer dump` locally before running tests.